src/manifest.rs

```diff
@@ -152,6 +152,23 @@ where
         .map(|x| x.into_iter().map(|(k, v)| (k, v.into())).collect())
 }
 
+/// Deserializes the `workspace` key as an `Option<bool>`, raising
+/// a hard error if `workspace = false` is explicity set, which is
+/// an invalid configuration.  Returns a `bool` whether the workspace
+/// key was explicitly set (`workspace = true`, return true) or omitted (false).
+fn deserialize_workspace_key<'de, D>(de: D) -> Result<bool, D::Error>
+where
+    D: serde::Deserializer<'de>,
+{
+    let option = Option::<bool>::deserialize(de)?;
+
+    match option {
+        Some(true) => Ok(true),
+        None => Ok(false),
+        Some(false) => Err(serde::de::Error::custom("`lints.workspace = false` is not valid configuration.  Either set `lints.workspace = true` or omit the key entirely.")),
```

Comment:
This isn't written down explicitly (and probably should be) but we generally single-space all our text, especially user-facing messages.
```suggestion
        Some(false) => Err(serde::de::Error::custom("`lints.workspace = false` is not valid configuration. Either set `lints.workspace = true` or omit the key entirely.")),
```